modules/tinymce/src/core/main/ts/fmt/MergeUtils.ts

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
I split this out as I needed a way to use specific functions from `MergeFormats` in `RemoveFormat`, however `MergeFormats` already references `RemoveFormat`, so using it directly would have created a circular dependency. The only changes I made to it was to use arrow functions.